frameworks/cassandra/docs/service-settings.md

```diff
@@ -43,4 +43,41 @@ You can configure whether the creation, transfer, and restoration of backups occ
 *   **DC/OS web interface**: `BACKUP_RESTORE_STRATEGY`: `string`
 
 # Virtual networks
+
 The Cassandra service can be run on a virtual network such as the DC/OS overlay network, affording each node its own IP address (IP per container). For details about virtual networks on DC/OS see the [documentation](/latest/networking/virtual-networks/#virtual-network-service-dns). For the Cassandra service, using a virtual network means that nodes no longer use reserved port resources on the Mesos agents.  This allows nodes to share machines with other applications that may need to use the same ports that Cassandra does. That means, however, that we cannot guarantee that the ports on the agents containing the reserved resources for Cassandra will be available, therefore we do not allow a service to change from a virtual network to the host network. **Once the service is deployed on a virtual network it must remain on that virtual network**. The only way to move your data to Cassandra on the host network is through a migration.
+
+# TLS
+
+It is possible to launch the Cassandra service over a TLS encrypted connection. Enabling TLS will switch Cassandra inter-node communication to TLS connections between all the Cassandra nodes. Cassandra will be still available on the same configured client port (default: `9042`).
+
+Enabling the TLS is possible only in `permissive` and `strict` cluster security modes. Both modes require a [service account](https://docs.mesosphere.com/service-docs/cassandra/cass-auth/). Additionally, a service account must have the `dcos:superuser` permission. If the permission is missing the Cassandra scheduler will not abe able to provision TLS artifacts.
+
+*   **In DC/OS CLI options.json**: `tls`: boolean (default: `false`)
+*   **DC/OS web interface**: `TASKCFG_ALL_CASSANDRA_ENABLE_TLS`: `boolean`
+
+To enable support for both TLS encrypted and non-TLS plaintext connections set the `tls_allow_plaintext` option to `true`. This option is disabled by default, so when the TLS is enabled, the non-encrypted connections would get refused.
+
+*   **In DC/OS CLI options.json**: `tls_allow_plaintext`: boolean (default: `false`)
+*   **DC/OS web interface**: `TASKCFG_ALL_CASSANDRA_ALLOW_PLAINTEXT`: `boolean`
+
+## Clients
+
+Clients connecting to the Cassandra service are required to use [the DC/OS CA bundle](https://docs.mesosphere.com/1.10/networking/tls-ssl/get-cert/) to verify the TLS connections.
+
+## cqlsh
+
+The `cqlsh` TLS can be configured by a [ssl section](https://github.com/apache/cassandra/blob/652d9f64f14d8375a8412561271a7abf27722f20/conf/cqlshrc.sample#L103) in the `cqlshrc` file. Connecting over TLS requires passing the `--ssl` flag.
```

Comment:
Change to: TLS can be configured for `cqlsh` in the [ssl section] of the `cqlshrc` file.